brands/shop/beauty.json

```diff
@@ -46,15 +62,16 @@
       "shop": "beauty"
     }
   },
-  "shop/beauty|Ulta Beauty": {
+  "shop/beauty|The Skin Bar at Ulta Beauty": {
     "countryCodes": ["us"],
-    "matchNames": ["ulta"],
+    "matchNames": ["ulta", "ulta skin bar"],
```

Comment:
the ulta match probably shouldn't be there with this so we don't match on the whole stores which are usually tagged just ulta

Reply:
I removed the extra matchNames in 71809d11